labels.mss

```diff
@@ -629,6 +521,7 @@
   shield-face-name: @sans;
   shield-clip: @shield-clip;
   shield-file: url("symbols/shields/[type]_[width]x[height].svg");
+  shield-transform: "scale(0.75)";
```

Comment:
I'd rather edit the script https://github.com/cyclosm/cyclosm-cartocss-style/blob/master/scripts/generate_shields.py to generate shield the right size. I'll handle this after this PR has been merged.

Reply:
I tried, I failed 😭